lib/robots/dor_repo/base.rb

```diff
@@ -13,6 +13,17 @@ def seed_uri(druid)
         Dor::Services::Client.object(druid).find.label
       end
 
+      def original_site_uri(druid)
+        cocina_object = Dor::Services::Client.object(druid).find
+        uri_note = cocina_object.description.note.find { |note| note.displayLabel == 'Original site' }
+        # if no original site note, use title since this is likely the first time generating
```

Comment:
Is it worth noting a bit more here why (from a workflow perspective) we have to hunt and peck around in the notes, title and label?

Reply:
Added some comments to provide insight into the workflow and moved method to thumbnail_generator

Reply:
Thanks for these helpful comments!